bin/heroku_postdeploy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# This file is being executed once after first deployment at heroku.
+set -e
+set -x
+
+# Compile static assets
+source ./bin/post_compile
+
+CELERY_ALWAYS_EAGER="True" ./manage.py sync_projects --no-commit
```

Comment:
I added this line t to be sure that sync_projects is executed during the build phase and to avoid strange timing errors when e.g. user gets to the main page sooner that sync ends.